Objects/bytesobject.c

```diff
@@ -819,7 +819,7 @@ _PyBytes_FormatEx(const char *format, Py_ssize_t format_len,
             if (v == NULL)
                 goto error;
 
-            if (fmtcnt < 0) {
+            if (fmtcnt == 0) {
                 /* last writer: disable writer overallocation */
```

Comment:
There is a typo in the original code: it should be "last write". Would you mind to fix it as well?

Reply:
No problem.